examples/digital_out/cont_gen_wfm_int_clk.py

```diff
@@ -0,0 +1,20 @@
+"""Example for generating digital signal.
```

Comment:
Use an article ("generating a digital signal") or make "signal" plural ("generating digital signals").

Reply:
Noted. It's updated.